go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/matcornic/hermes/v2 v2.1.0
 	github.com/pelletier/go-toml v1.9.5
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.15.0
+	github.com/prometheus/client_golang v1.15.1
```

Comment:
noise: Can we drop the updates from this PR, or were they required?

Reply:
Most likely not required, I'll drop them